data/RanaMuskan.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Rana Muskan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Contributor, Youtuber, love to code.",
+  "avatar": "https://github.com/RanaMuskan.png",
```

Comment:
you can add tags like this;
![image](https://user-images.githubusercontent.com/88339569/215161070-dad2d630-0fad-4de8-bb79-f747399fcbae.png)

Reply:
I have changed the code. Thanks 😊